### DISCUSSION TITLE
Season of Docs 2022

### DISCUSSION THREAD

**Discussion by capsulecorplab** [PolyMathOrg/PolyMath#228] [General]
[Season of Docs](https://developers.google.com/season-of-docs) is a Google sponsored program that connects open-source organizations with technical writers to improve documentation. Applications for open-source organizations are open between February 23 - March 25 for 2022, as per their [timeline](https://developers.google.com/season-of-docs/docs/timeline). Could be a good opportunity for PolyMath to validate its documentation.

**Comment by SergeStinckwich** [comment]
Yes would be great. Are you volunteer to take care about that? Thank you.

**Reply by capsulecorplab** [reply]
I think someone more familiar with PolyMath's documentation roadmap and current needs would be better suited to submit an application, but I'd be happy to help review the application!
